testing/auth.ts

```diff
@@ -130,6 +133,36 @@ class CredentialHandler {
     print('Credentials updated!');
   }
 
+  handleCustom(
+    paramDefs: Array<{
+      name: string;
+      description: string;
+    }>,
+  ) {
+    assertCondition(this._authDef.type === AuthenticationType.Custom);
+    if (paramDefs.length === 0) {
+      printAndExit(
+        `Please define one or more entries for "params" in the defaultAuthentication section of this Pack definition.`,
```

Comment:
Nit, this can be used with systemAuthentication too, right?

Reply:
oh yeah hmm this is what all of our things do right now. I can update them all to reference the authentication set in the pack definition, esp. since we swtiched over to the declarative non-manifest version for our examples

Reply:
Ah yeah that's confusing, let's just clean them all up to say something more general about the authentication configuration in the pack defintion.